spr/spr.go

```diff
@@ -461,7 +461,8 @@ func (sd *stackediff) mustgit(argStr string, output *string) {
 
 func check(err error) {
 	if err != nil {
-		panic(err)
+		fmt.Printf("error: %s\n", err)
```

Comment:
I find the panic stack trace very useful for debugging, but agree that at this point exposing it to the end user is a bit harsh. Can we add a debug flag to print the stack trace when running with debug. 

Instead of passing around a cfg object everywhere, in the case of an error you can use the following to the the cfg and check for debug:
```
gitcmd := realgit.NewGitCmd(config.DefaultConfig())
cfg := config.ParseConfig(gitcmd)
```

also in spr/main.go there is a debug cmd line arg that can set the debug config.

Reply:
I went with an environment variable to avoid putting complex logic like parsing a config file into the error handling path (what if _that_ fails?)

Reply:
This works and does avoid complicated stuff in the error handling path. 
If we set the SPR_DEBUG variable in spr/main.go in the Before func around line 107 where is parses the debug flag, then the debug flag will work as well.